geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/InternalGfshCommand.java

```diff
@@ -66,4 +68,11 @@ public Gfsh getGfsh() {
   public ManagementService getManagementService() {
     return ManagementService.getExistingManagementService(getCache());
   }
+
+  @Override
+  public InternalConfigurationPersistenceService getConfigurationPersistenceService() {
```

Comment:
Other than the return type, this is verbatim-equivalent to the overridden method.

If the desire is to avoid the abundance of casts from the public to the internal that exist throughout the various gfsh commands, then I would be inclined to push this to a different ticket.  If we want to remove the casts, we should remove those casts in the same commit that changes the return type, if only just for cleanliness's sake.

Unless I'm missing it, it doesn't look like this method is used anywhere beyond your new command that didn't already have a cast to the `Internal*` version, making the parent method equivalent in the current implementation.

If you do want to keep this method and leave the removal of the now-redundant casts for a future effort, I might prefer this method as the following:

```
  @Override
  public InternalConfigurationPersistenceService getConfigurationPersistenceService() {
    return (InternalConfigurationPersistenceService) super.getConfigurationPersistenceService();
  }

```

Reply:
this is fixed as suggested.